### PR TITLE
Delete Batch Storybook Component

### PR DESCRIPTION
**Pull request by Ekamba**

### CODE REVIEW

**Comment by Ekamba** [comment]
@bytewiz Can you have a look on my PR, if I have something to change? Thanks

File: src/client/components/Delete_Comfim/ModalList.component.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '../button/button.component';
+
+const ModalList = ({ isShowing, hide }) =>
+  isShowing
```
Comment (senner008) [inline src/client/components/Delete_Comfim/ModalList.component.js]: I suggest decomposition. At least 2-3 subcomponents

Comment (Ekamba) [reply src/client/components/Delete_Comfim/ModalList.component.js]: Hi senner008, thank you for your review. Can you tell more details about what I have to do so this component be perfect?

Comment (senner008) [reply src/client/components/Delete_Comfim/ModalList.component.js]: Well, what does it contain?

**Pull request closed without merging**